src/modules/animalrescue.mainmodule/animalrescue.mainmodule.helpers/Interfaces/IsPet.cs

```diff
@@ -0,0 +1,8 @@
+namespace animalrescue.mainmodule.helpers.interfaces
+{
+    public interface IsPet
```

Comment:
rename class + file to `IIsPet` or this might be better: `IHasPetCharacteristics`, probably more descriptive

Reply:
renamed to IHasPetCharacteristics
![image](https://user-images.githubusercontent.com/4991909/203627426-a1b78cb5-6d67-42b6-ade3-a188dfbae05e.png)